bluezero/adapter.py

```diff
@@ -258,10 +259,13 @@ def _properties_changed(self, interface, changed, invalidated, path):
         Handle DBus PropertiesChanged signal and
         call appropriate user callback
         """
+        macaddr = path.split("/")[-1].replace("dev_", '').replace("_", ":")
         if self.on_disconnect is not None:
             if 'Connected' in changed:
                 if not changed['Connected']:
-                    self.on_disconnect()
+                    self.on_disconnect(macaddr)
```

Comment:
I need to think about this because this will break anyone's code that has an `on_disconnect` callback.
If we are going to pass a parameter, rather than passing just the device mac address to the `on_disconnect` callback, would it be better to pass a device object?
The advantage of a device object would be that people have access to all the properties of that device.
An example of what this might look like is in the `_interfaces_added` function for `on_device_found`.

Reply:
Actually was the first thing I thought but I didn't find a way to create the device because i guess th ctor wants the device ("hci") as well and in that point I didn't have.
If you have a more elegant way , please share it and i'll make it.

Reply:
BTW Ill' check _interfaces_added and on_device_found

Reply:
done